childprocess.gemspec

```diff
@@ -20,6 +20,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.4.0'
 
+  s.add_runtime_dependency "ffi", "~> 1.0", ">= 1.0.11" if Gem.win_platform?
```

Comment:
This won't do what you want. Gemspecs are evaluated at build time, so this will only work if the gem is built on a Windows machine.

We don't include the `ffi` gem intentionally, as the vast majority of users don't need it and it slows down installs due to the time it takes to compile native extensions.

Reply:
hmm, what about `RbConfig::CONFIG['target_os'] =~ /mingw32|mswin/` then? It seems strange to require gem, but not have it in deps.

Reply:
We used to do this. Please see #160 for why we removed. There are many good reasons to not do this.